src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/OrderItem.orm.xml

```diff
@@ -17,6 +17,11 @@
                                       http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
 
     <mapped-superclass name="Sylius\Component\Core\Model\OrderItem" table="sylius_order_item">
+        <field name="immutableProductName" column="immutable_product_name" type="string" nullable="true"/>
```

Comment:
I'm not sure if the product name on order item should be nullable. It is pretty important data